src/ImageSharp/Formats/Webp/Lossy/LossyUtils.cs

```diff
@@ -146,6 +178,104 @@ public static int Vp8_SseNxN(Span<byte> a, Span<byte> b, int w, int h)
             return count;
         }
 
+#if SUPPORTS_RUNTIME_INTRINSICS
+        [MethodImpl(InliningOptions.ShortMethod)]
+        private static int Vp8_Sse16xN_Sse2(Span<byte> a, Span<byte> b, int numPairs)
+        {
+            Vector128<int> sum = Vector128<int>.Zero;
+            int offset = 0;
```

Comment:
```suggestion
            nint offset = 0;
```
😉 (to avoid the `movsxd` (sign extension)).

For `offset + WebpConstants.Bps` I'd expect the JIT knows how to handle the constants, so will convert them to `nint`s too.